### ISSUE TITLE
Typescript types seem missing

### ISSUE THREAD

**Issue by arendjantetteroo** [currents-dev/cypress-cloud#66]
### Before opening, please confirm:

- [X] I have searched for duplicate or closed issues](https://github.com/currents-dev/cypress-cloud/issues) and [discussions](https://github.com/currents-dev/cypress-cloud/discussions).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.
- [X] I acknowledge that I will attach a **full debug log**, otherwise the issue will be closed with no response.

### Environment information

<details>

```
# Put output below this line


```

</details>

### Describe the bug

Typescript can't seem to find the types for the plugin:
https://github.com/currents-dev/cypress-cloud/discussions/65#discussioncomment-5253571

### Expected behavior

No errors for typescript. 

### Command and Setup

Follow the installation steps where cypress is configured to use typescript for its config files. 
https://github.com/currents-dev/cypress-cloud#setup

where cypress uses cypress.config.ts

### Full log and debug output

<details>

```
// Put your logs below this line


```

</details>

**Comment by agoldis** [comment]
Should be fixed in [1.3.](https://github.com/currents-dev/cypress-cloud/releases/tag/v1.4.2)

**Comment by arendjantetteroo** [comment]
Thanks for the quick fix. 

However, I now have a different error:

I don't use the const require indicated in the documentation, but use `import {cloudPlugin} from 'cypress-cloud/plugin';` 

I now get
```
TS7016: Could not find a declaration file for module 'cypress-cloud/plugin'. '/node_modules/.pnpm/cypress-cloud@1.4.3/node_modules/cypress-cloud/plugin/index.js' implicitly has an 'any' type.<br/>Try `npm i --save-dev @types/cypress-cloud` if it exists or add a new declaration (.d.ts) file containing `declare module 'cypress-cloud/plugin';`
```
